src/main/java/de/tum/in/www1/exerciseapp/service/CourseService.java

```diff
@@ -60,9 +66,52 @@ public Course save(Course course) {
                 || user.getAuthorities().contains(adminAuthority)
         );
         List<Course> userAuthorizedCourses = userCourses.collect(Collectors.toList());
+        userAuthorizedCourses = moveTutorialToTop(userAuthorizedCourses);
         return userAuthorizedCourses;
     }
 
+    /**
+     * Rearange Course List so Tutorial is always first course
+     *
+     * @return same list reordered so tutorial is first item.
+     */
+    private List<Course> moveTutorialToTop(List<Course> allCourses){
+        List<Course> temp = new ArrayList<>();
+        List<Course> reorderedList = new ArrayList<>();
+        for(Course course : allCourses){
+            if(course.getTitle().equals("Tutorial")){
+                reorderedList.add(course);
+            }else{
+                temp.add(course);
+            }
+        }
+        reorderedList.addAll(temp);
+        return reorderedList;
+    }
+
+    /**
+     * Initalize the tutorial with the partcipation and result for a User
+     *
+     * @param user the user for whom the participation and result is supposed to be added
+     */
+    @Transactional(readOnly = false)
+    public void initTutorial(User user){
+        Optional<Course> course = courseRepository.findOneByTitle("Tutorial");
+
+        if(course.isPresent()) {
+            Course tutorial = course.get();
+            log.debug("Starting to initalize tutorial for {}", user);
+
+            Optional<Exercise> exercise = exerciseRepository.findOneByTitle("Exercise 1");
```

Comment:
I'd like to make it not dependent on the name of the exercise but I figured Id would come with the same problem. Is the name fine or should it be different, adding the dependency on CourseId later on.